### ISSUE TITLE
Error processing transaction request: rlp: input list has too many elements for types.LegacyTx

### ISSUE THREAD

**Issue by imdsc** [web3j/web3j#1404]
# _Issue_title_
_A clear and concise description of what the issue is._
Error processing transaction request: rlp: input list has too many elements for types.LegacyTx

## _Issue_description_
_Please describe what are you trying to do and why it doesn't work_

![image](https://user-images.githubusercontent.com/38461516/116682828-542e7a80-a9e1-11eb-9db5-0792cd9906c2.png)

## _Issue_context_
_Please provide additional information that might be helpful_
how to resolve it

**Comment by mohamedelshami** [comment]
Hi @imdsc could you please attach the code snippet so that we can try and reproduce this error?

**Comment by imdsc** [comment]
@Test
    public void transferTest() {
        Web3j web3 = Web3j.build(new HttpService(NODE_URL));
        try {
            Credentials credentials = WalletUtils.loadCredentials(WALLET_PASSWORD, WALLET_FILE_PATH);
            String v = web3.web3ClientVersion().send().getWeb3ClientVersion();
            TransactionReceipt transactionReceipt = Transfer.sendFundsEIP1559(
                    web3, credentials, "0x5F7e154537CeEFEe8906B2d0d2D72a9251BC1bef",
                    BigDecimal.valueOf(2),
                    Convert.Unit.ETHER,
                    DefaultGasProvider.GAS_LIMIT,
                    DefaultGasProvider.GAS_PRICE,
                    DefaultGasProvider.GAS_LIMIT)
                    .send();

        } catch (Exception e) {
            e.printStackTrace();
        }
    }

**Comment by imdsc** [comment]
```java
 @Test
    public void transferTest() {
        Web3j web3 = Web3j.build(new HttpService(NODE_URL));
        try {
            Credentials credentials = WalletUtils.loadCredentials(WALLET_PASSWORD, WALLET_FILE_PATH);
            String v = web3.web3ClientVersion().send().getWeb3ClientVersion();
            TransactionReceipt transactionReceipt = Transfer.sendFundsEIP1559(
                    web3, credentials, "0x5F7e154537CeEFEe8906B2d0d2D72a9251BC1bef",
                    BigDecimal.valueOf(2),
                    Convert.Unit.ETHER,
                    DefaultGasProvider.GAS_LIMIT,
                    DefaultGasProvider.GAS_PRICE,
                    DefaultGasProvider.GAS_LIMIT)
                    .send();

        } catch (Exception e) {
            e.printStackTrace();
        }
    }

```

**Comment by imdsc** [comment]
geth version:Geth/v1.10.2-stable-97d11b01/windows-amd64/go1.16
web3j-core version: 4.8.4

**Comment by imdsc** [comment]
@mohamedelshami

**Comment by imdsc** [comment]
![Uploading image.png…]()

**Comment by imdsc** [comment]
![image](https://user-images.githubusercontent.com/38461516/116687030-25b39e00-a9e7-11eb-945e-ddea7edf0002.png)
maybe the reason is gasPrice is null

**Comment by mohamedelshami** [comment]
I don't think so, BaseFee or FeePremium is a substitute for gasPrice in EIP1559.

**Comment by ucwong** [comment]
Is this same problem
```
Error processing request: invalid argument 0: json: cannot unmarshal non-string into Go value of type common.Hash
```

**Comment by AlexandrouR** [comment]
@imdsc Hi from my understanding Geth has not implemented EIP 1559 yet

**Comment by cristianberbecaru** [comment]
Hi,

I am unable to send transactions. They are successfully sent to blockchain, but with gas price 0, so the transactions won't be mined. I am using Geth 1.10.4.

I tried the following code:

 RawTransaction transaction1559 = RawTransaction.createEtherTransaction(
                ChainId.ROPSTEN,
                BigInteger.valueOf(4L),
                Transfer.GAS_LIMIT,
                transfer.getRecipientAddress(),
                amountInWei,
                DefaultGasProvider.GAS_LIMIT,
                DefaultGasProvider.GAS_PRICE
        );
 EthSendTransaction ethSendTransaction = manager.signAndSend(transaction1559);
